wiki/Beginner/en.md

```diff
@@ -47,104 +49,99 @@ Touchscreen tends to struggle with:
 
 ### Keyboard Tapping
 
-For mouse and tablet or any other input device, you may also use your keyboard (Z and X by default) to hit the circles instead. The benefit of doing so is spreading the physical movement across two hands, rather than doing the clicking with your aiming device, thus allowing you to more easily hit faster sections of notes. This takes some time to get used to, but if you decide on using your keyboard to tap circles, it's a good idea to get used to it early on.
+For mouse and tablet or any other input device, you may also use your keyboard (`Z` and `X` by default) to hit the circles instead. The benefit of doing so is spreading the physical movement across two hands, rather than doing the clicking with your aiming device, thus allowing you to more easily hit faster sections of notes. This takes some time to get used to, but if you decide on using your keyboard to tap circles, it's a good idea to get used to it early on.
 
 Gameplay Techniques
 -------------------
 
 ### Accuracy and Timing
 
-One of the most important factors when playing osu! is your timing, much like other rhythm games! In this sense, if you've listened to a lot of music, it should be fairly easy to pick up on. The goal of accuracy and timing is to tap the notes at the correct time as precisely as possible.
+One of the most important factors when playing *osu!* is your timing, much like other rhythm games! In this sense, if you've listened to a lot of music, it should be fairly easy to pick up on. The goal of accuracy and timing is to tap the notes at the correct time as precisely as possible.
 
 Each object in the game has an approach circle; a large ring that slowly approaches the hit circle itself. When the ring reaches the circle, it's time to tap it! If you pay attention to them carefully, it will help you recognize whether some notes are on half-beats or full beats, or whether you need to wait for the next note.
 
 Music works in beats and a rhythm. When you start a map (or even before you play) pay attention to the how the song sounds, and you can get a pretty good idea on the melody that you will be hitting circles to. Getting along with the music more closely will help you out in achieving better timing and coordination over it.
```

Comment:
"Music works in beats and rhythms. When you start a map (or even before you play), pay attention..."

(Get rid of the "a" and pluralize rhythms; add a comma after the parentheses.)